game/src/layer/pandemic.rs

```diff
@@ -231,7 +231,8 @@ fn make_controls(
         ]),
     ];
 
-    col.push(Checkbox::text(
+    // MJK: how to show this UI?
```

Comment:
Ooh, it's undocumented. Run with `--pandemic`, then it'll show up in layers. This was some work by @omalaspinas that's on hold.

Reply:
done!